pkgs/applications/misc/swaysettings/default.nix

```diff
@@ -0,0 +1,74 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, accountsservice
+, appstream-glib
+, desktop-file-utils
+, gettext
+, glib
+, gobject-introspection
+, pantheon
+, gtk-layer-shell
+, gtk3
+, json-glib
+, libgee
+, libhandy
+, libxml2
+, meson
+, ninja
+, pkg-config
+, python3
+, sway
+, vala
+, wrapGAppsHook
+, xkeyboardconfig
+}:
+
+stdenv.mkDerivation rec {
+  pname = "swaysettings";
+  version = "2022-05-25";
+
+  src = fetchFromGitHub {
+    owner = "ErikReider";
+    repo = "SwaySettings";
+    rev = "c60a235061faf68ca1cf24d22b2aed73eb31e6c0";
+    hash = "sha256-EVGY4LbiH+RB56sYGOkYCdOc01LqbJIg9mJjgvFd4oY=";
+  };
+
+  nativeBuildInputs = [
+    appstream-glib
+    desktop-file-utils
+    gettext
+    meson
+    ninja
+    pkg-config
+    python3
+    vala
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
```

Comment:
I think gsettings-desktop-schemas is required for `org.gnome.desktop.interface` schemas

https://github.com/ErikReider/SwaySettings/blob/c60a235061faf68ca1cf24d22b2aed73eb31e6c0/src/Pages/Themes/Themes_Page.vala#L5
https://nixos.org/manual/nixpkgs/unstable/#ssec-gnome-packaging

Reply:
Good. I fixed that.